src/bin/travis/testSbt.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
works for me. I know that `/bin/bash` isn't generally considered the best thing to do so I just assumed the docs I saw were doing things for a good reason ... however I realize that generally things are the way they are just because that's how they are :)